core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/StoreCallbacksImpl.scala

```diff
@@ -522,7 +522,7 @@ object StoreCallbacksImpl extends StoreCallbacks with Logging with Serializable
         val columnNullable = context.getColumnNullable
         logDebug(s"StoreCallbacksImpl.performConnectorOp alter table ")
         session.alterTable(tableName, addOrDropCol, StructField(columnName,
-          CatalystSqlParser.parseDataType(columnDataType), columnNullable))
+          CatalystSqlParser.parseDataType(columnDataType), columnNullable), defaultValue = "")
```

Comment:
smart connector route also needs to add defaultValue argument to the procedure

Reply:
Corrected the code